.gitlab-ci.yml

```diff
@@ -86,6 +88,12 @@ default:
     - /^pre-v[0-9]+\.[0-9]+-[0-9a-f]+$/
     - web
 
+.chaos-only:                       &chaos-only
+  only:
+    variables:
```

Comment:
you will need `&default-vars` here.

Reply:
Not sure if i understood when to add ```&default-vars```. Could you please do a suggested change?